_posts/2016/11/2016-11-21-gap-lesson.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+authors: ["Alexander Konovalov"]
+title: "Programming with GAP"
+date: 2016-11-18
+time: "15:00:00"
+category: ["Lessons"]
+---
+
+This is a blog post about the Software Carpentry lesson
+["Programming with GAP"](http://alex-konovalov.github.io/gap-lesson/).
```

Comment:
This sentence feels a bit lonely on its own - maybe something like, "Software Carpentry is more than just a set of workshops and lessons. It is also a way to *develop* lessons, one that we have used successfully to create a lesson on [Programming with GAP](...)."